modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,246 @@
+      }
+      rhcs = {
+        version = ">= 1.3.0"
+        source  = "terraform-redhat/rhcs"
+      }
+    }
+  }
+
+  data "rhcs_policies" "all_policies" {}
+
+  data "rhcs_versions" "all" {}
+
+  module "create_account_roles" {
+    source  = "terraform-redhat/rosa-sts/aws"
+    version = "0.0.15"
+
+    create_operator_roles = false
+    create_oidc_provider  = false
+    create_account_roles  = true
+
+    account_role_prefix    = var.account_role_prefix
+    rosa_openshift_version = var.openshift_version
+    account_role_policies  = data.rhcs_policies.all_policies.account_role_policies
+    operator_role_policies = data.rhcs_policies.all_policies.operator_role_policies
+    all_versions           = data.rhcs_versions.all
+    tags                   = var.tags
+  }
+EOF
+----
+
+.. You define the account role prefix structure in the `output.tf` file. This output definition allows you to specify how the various generated roles are constructed. Run the following command to create your `output.tf` file:
++
+[source,terminal]
+----
+$ cat<<-EOF>output.tf 
+  output "account_role_prefix" {
+    value = module.create_account_roles.account_role_prefix
+  }
+EOF  
+----
+
+.. The `variables.tf` allows you to specify values you want for select variables. If you exported a variable for the `account_role_prefix` earlier, leave this variable's default value blank. Setting the variable in both places with different values can produce unexpected results. Run the following command to create your `variables.tf` file:
++
+[IMPORTANT]
+====
+Do not include your OCM token in this file if it is not stored in a safe location.
+====
++
+[source,terminal]
+----
+$ cat<<-EOF>variables.tf 
+  variable "openshift_version" {
+    type = string
+    default = "4.13"
+    description = "Enter the desired OpenShift version as X.Y. This version should match what you intend for your ROSA cluster. For example, if you plan to create a ROSA cluster using '4.13.10', then this version should be '4.13'. You can see the supported versions of OpenShift by running 'rosa list version'."
+  }
+
+  variable "account_role_prefix" {
+    type    = string
+    default = ""
+    description = "Your account roles are prepended with whatever value you enter here. The default value in the ROSA CLI is 'ManagedOpenshift-' before all of your account roles."
+  }
+
+  variable "tags" { <1>
+    type        = map
+    default     = null
+    description = "(Optional) List of AWS resource tags to apply."
+  }
+EOF
+----
++
+--
+<1> The `tags` parameter uses a map of strings variable. The format that it takes looks like the following example:
++
+[source,terraform]
+----
+variable "tags" {
+  type    = "map"
+  default = {
+    "us-east-1" = "image-1234"
+    "us-west-2" = "image-4567"
+  }
+}
+----
+--
+. In the directory where you saved these Terraform files, run the following command to set up Terraform to create these resources:
++
+[source,terminal]
+----
+$ terraform init
```

Comment:
I did get a timeout warning running this the first time, but the second time was fine:

```
│ Error: Failed to install provider
│ 
│ Error while installing hashicorp/random v3.5.1: releases.hashicorp.com: Get
│ "https://releases.hashicorp.com/terraform-provider-random/3.5.1/terraform-provider-random_3.5.1_linux_amd64.zip":
| dial tcp: lookup releases.hashicorp.com on 127.0.0.53:53: read udp 127.0.0.1:37618->127.0.0.53:53: i/o timeout
```

Reply:
@nirarg what would trigger this error? It appears like a network issue?